Broken CI on windows runner

CI is broken on windows runner.

```
DEPRECATION: Python 3.5 reached the end of its life on September [13](https://github.com/ChanTsune/list-reserve/runs/5382096073?check_suite_focus=true#step:4:13)th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.
Requirement already satisfied: pip in c:\hostedtoolcache\windows\python\3.5.4\x64\lib\site-packages (20.3.4)
Collecting pip
  Using cached pip-20.3.4-py2.py3-none-any.whl (1.5 MB)
  Downloading pip-20.3.3-py2.py3-none-any.whl (1.5 MB)
running install
running bdist_egg
running egg_info
creating list_reserve.egg-info
writing dependency_links to list_reserve.egg-info\dependency_links.txt
writing top-level names to list_reserve.egg-info\top_level.txt
writing list_reserve.egg-info\PKG-INFO
writing manifest file 'list_reserve.egg-info\SOURCES.txt'
C:\hostedtoolcache\windows\Python\3.5.4\x64\lib\distutils\dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
  warnings.warn(msg)
package init file 'list_reserve\__init__.py' not found (or not a regular file)
reading manifest file 'list_reserve.egg-info\SOURCES.txt'
writing manifest file 'list_reserve.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build
creating build\lib.win-amd64-3.5
creating build\lib.win-amd64-3.5\list_reserve
copying list_reserve\py.typed -> build\lib.win-amd64-3.5\list_reserve
copying list_reserve\__init__.pyi -> build\lib.win-amd64-3.5\list_reserve
running build_ext
building 'list_reserve' extension
error: Microsoft Visual C++ [14](https://github.com/ChanTsune/list-reserve/runs/5382096073?check_suite_focus=true#step:4:14).0 is required. Get it with "Microsoft Visual C++ Build Tools": http://landinghub.visualstudio.com/visual-cpp-build-tools
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -31,7 +31,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        python -m pip install --upgrade pip setuptools
         python setup.py install
     - name: Test
       run: |

